pandas/core/arrays/masked.py

```diff
@@ -160,6 +161,9 @@ def _empty(cls, shape: Shape, dtype: ExtensionDtype):
             )
         return result
 
+    def _formatter(self, boxed: bool = False) -> Callable[[Any], str | None]:
+        return super()._formatter(boxed=True)
```

Comment:
we're ignoring the boxed keyword?

Reply:
Yeah since I think we'll need to always return str given the NEP 51 change. Actually probably just simpler to `return str` here